plugins/modules/influxdb_retention_policy.py

```diff
@@ -59,7 +59,7 @@
     default: false
   shard_group_duration:
     description:
-      - Determines the time range covered by a shard group. If specified it must be at least one hour. If none, it's determined
+      - Determines the time range covered by a shard group. If specified it must be at least one hour. If none, it is determined
```

Comment:
Mixed feelings on this one, but maybe:
```suggestion
      - Determines the time range covered by a shard group. If specified it must be at least one hour. If V(null), it is determined
```
?
Or "If none is passed"...  to ensure no one tries `shard_group_duration: none`.

Overkill?

Reply:
How about "If not provided, ..."?

Reply:
Yup, definitely better

Reply:
```suggestion
      - Determines the time range covered by a shard group. If specified it must be at least one hour. If not provided, it is determined
```